Source/CarthageKit/Xcode.swift

```diff
@@ -1283,7 +1283,7 @@ private func stripArchitecture(frameworkURL: NSURL, architecture: String) -> Sig
 }
 
 /// Returns a signal of all architectures present in a given framework.
-public func architecturesInFramework(frameworkURL: NSURL) -> SignalProducer<String, CarthageError> {
+private func architecturesInFramework(frameworkURL: NSURL) -> SignalProducer<String, CarthageError> {
```

Comment:
This is also used in `XcodeSpec.swift`, and might be used publically because this is a public API of `CarthageKit.framework`. So we should preserve this as public, I think.